File: src/app/src/main/java/com/example/followme/Retrofit/RetrofitService.java
package com.example.followme.Retrofit;

import com.google.gson.Gson;
import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitService {
    private Retrofit retrofit;

    public RetrofitService() {
        initializeRetrofit();
    }

    private void initializeRetrofit() {
        retrofit = new Retrofit.Builder();
                .baseUrl("http://iseproject07e.informatik.htw-dresden.de:3306")
                .addConverterFactory(GsonConverterFactory.create(new Gson()))
                .build();
    }
}

File: src/app/src/main/java/com/example/followme/Retrofit/RetrofitService.kt
package com.example.followme.Retrofit

import com.google.gson.Gson
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

/**
 * Klasse Für die Konverteirung der Daten von JSON zu Arrays, siehe Retrofit Library
 */
class RetrofitService {
    private lateinit var retrofit: Retrofit

    init {
        initializeRetrofit()
    }

    private fun initializeRetrofit() {
        retrofit = Retrofit.Builder()
                //Url wird geadded, wo es konvertiert werden soll, HTW-Server-Port
            .baseUrl("http://iseproject07e.informatik.htw-dresden.de:8080")
                //JSon Libary von Google wird benutzt G-Son, witziger Name Hahaha
            .addConverterFactory(GsonConverterFactory.create(Gson()))
            .build()
    }

    fun getRetrofit(): Retrofit {
        return retrofit
    }
}